From e8951c9af1c8713d75f4d7d35a2cb52cbe1149ec Mon Sep 17 00:00:00 2001
From: Sun, Mingruo <mingruo.sun@intel.com>
Date: Fri, 19 Apr 2013 22:30:20 +0800
Subject: [PORT FROM MAIN] Update the vsp VP8 header to VIED 1.6[Video-MRFLD]

BZ: 101933

Update the vsp VP8 header to VIED 1.6 Release

Change-Id: Id6cc6f1a768c3fce7e8ad43e9289a27380ff4bfe
Orig-Change-Id: I41fc7daadfb5e8bcf69bd1fde78481e5d94b2fdf
Signed-off-by: Sun, Mingruo <mingruo.sun@intel.com>
Reviewed-on: http://android.intel.com:8080/102782
Reviewed-by: Zhang, Zhangfei <zhangfei.zhang@intel.com>
Reviewed-by: Yuan, Shengquan <shengquan.yuan@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Shi, PingX <pingx.shi@intel.com>
Tested-by: Shi, PingX <pingx.shi@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libc/kernel/common/linux/vsp_fw.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/libc/kernel/common/linux/vsp_fw.h b/libc/kernel/common/linux/vsp_fw.h
index 3cf9932..fbf46f4 100644
--- a/libc/kernel/common/linux/vsp_fw.h
+++ b/libc/kernel/common/linux/vsp_fw.h
@@ -212,7 +212,8 @@ struct VssVp8encEncodedFrame {
     uint32_t partition_id;
     uint32_t buffer_level;
     uint32_t quality;
-    uint32_t reserved;
+    uint32_t surfaced_of_ref_frame[3];
+    uint32_t reserved[15];
     uint32_t coded_data[1];
 };
 
@@ -221,6 +222,7 @@ struct VssVp8encEncodedFrame {
  */
 struct VssVp8encPictureParameterBuffer {
     struct VssProcPictureVP8 input_frame;
+    struct VssProcPictureVP8 recon_frame;
 
     uint32_t version;
     uint32_t pic_flags;
-- 
1.7.4.1

